corehq/apps/app_manager/suite_xml.py

```diff
@@ -1691,6 +1707,12 @@ def get_datum_meta_module(self, module, use_filter=False):
             })
         return datums
 
+    def get_userdata_autoselect(self, key, session_id, mode):
+        xpath = session_var(key, path='user/data')
+        datum = SessionDatum(id=session_id, function=xpath)
+        assertions = self.get_auto_select_assertions(xpath, mode, [key])
+        return datum, assertions
```

Comment:
I moved this from higher up, 'cause the only place this is used now is just below.  Think I should un-abstract it now that it's a one-off?